components/google-cloud/google_cloud_pipeline_components/experimental/custom_job/custom_job_sample.ipynb

```diff
@@ -0,0 +1,348 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# Copyright 2021 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Vertex Pipelines: Vertex AI Custom Job training wrapper\n",
+    "\n",
+    "## Overview\n",
+    "This notebook shows how to use the `run_as_vertex_ai_custom_job` wrapper to convert any component to run it as a Vertex AI custom job. This allows users to take advantage of the vertical and horizontal scaling for computation heavy tasks on Vertex AI. This requires that the underlying component has built in support distributed computation. To learn more about Vertex AI Custom Job see [Vertex AI Custom Training](https://cloud.google.com/vertex-ai/docs/training/custom-training). \n",
+    "\n",
+    "\n",
+    "## Vertex AI Custom Job Wrapper Interface\n",
```

Comment:
nit - I think the interface doc is already covered in 
https://github.com/kubeflow/pipelines/blob/5f4383e5c5fb51ae358806edb6671b73fc84390b/components/google-cloud/google_cloud_pipeline_components/experimental/custom_job/custom_job.py#L51
copying it here is fine but might be out of sync. 

Maybe just link to the source of truth

Reply:
good point! updated accordingly.